.travis.yml

```diff
@@ -89,7 +89,7 @@ script:
 - cd monkey_island/cc/ui
 - npm ci  # See https://docs.npmjs.com/cli/ci.html
 - eslint ./src --quiet  # Test for errors
-- JS_WARNINGS_AMOUNT_UPPER_LIMIT=4
+- JS_WARNINGS_AMOUNT_UPPER_LIMIT=8
```

Comment:
https://travis-ci.com/github/guardicore/monkey/builds/185225218

New warnings are mainly caused by usage of ` instead of ' for strings. 
This is a syntax which allows to display multiple classnames in a readable manner, so I don't want to remove it.